src/databricks/labs/ucx/framework/crawlers.py

```diff
@@ -41,6 +41,27 @@ def _schema_for(cls, klass):
             fields.append(f"{f.name} {spark_type}{not_null}")
         return ", ".join(fields)
 
+    @classmethod
+    def _filter_none_rows(cls, rows):
+        results = []
+        nullable_fields = []
```

Comment:
```suggestion
        nullable_fields = set()
```
so that the lookup is faster for 10k rows